lessons/Early-Math/evaluating-exponents/langs/en-us/index.adoc

```diff
@@ -0,0 +1,183 @@
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Translate between words, mathematical expressions, and Circles of Evaluation.
+* Evaluate expressions involving exponents and one other operation.
+* Evaluate expressions involving exponents and negations.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Circles of Evaluation to analyze the structure of expressions with exponents.
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Key Points For The Facilitator
+|
+* Ensure that students are comfortable evaluating exponents during the Launch. Circles of Evaluation will help students parse expressions that involve exponents.
+
+|===
+
+== Expressions with Exponents and One Operation @duration{30 minutes}
+
+=== Overview
+
+Sudents use Circles, words, and mathematical notation to analyze the structure of expressions with exponents and one operation.
+
+=== Launch
+
+[.lesson-instruction]
+- Show students the following expression: @math{2^3}
+- One way of reading this expression is "two raised to the power of three." Do you know any other ways of translating this expression into words?
+** _Two to the third power; two to the third; two cubed_
+- What does it mean to _cube_ a number?
+** We multiply it by itself three times (literally, compouting the volume of a cube with a side-length of that number)
+- What does this expression evaluate to?
+** @math{2^3 = 2 \times 2 \times 2 = 8}
+
+Clarify and reinforce some key concepts about exponents via class discussion:
+
+- The number on the left is called the @vocab{base}. That number is multiplied by itself when we apply the exponent.
+
+- The smaller, raised number after the base is called the @vocab{exponent}; it indicates how many times to multiply the base.
+
+- "Cubing" is the same as "raising to the third power", and "squaring" is the same as raising to the second power.
+
+- There is no special terminology for any other exponents.
+
+Explain to students that we can represent expressions with exponents as Circles of Evaluation. The function we use to do so is called `num-expt`; when we use `num-expt`, the first value is the _base_, and the second value is the exponent.
+
+[.lesson-point]
+Exponents are valuable because they act as a shorthand.
+
+The Circle of Evaluation for @math{2 \times 2 \times 2 \times 2 \times 2} looks like this:
+
+[.centered-image]
+@show{(coe '(* (* (* (* 2 2) 2) 2) 2))}
+
+Here is the Circle of Evaluation for @math{2^5}, an equivalent expression:
+
+[.centered-image]
+@show{(coe '(num-expt 2 5))}
+
+The Circle of evaluation with `num-expt` is a lot shorter, and easier to read!
+
+[.lesson-instruction]
+--
+- Take a look at the expressions and Circles below.
+
+[.embedded, cols="^.^1,^.^1", grid="none", stripes="none" frame="none"]
+|===
+|@math{5^2}							| @math{2^5}
+|@show{(coe  '(num-expt 5 2))}		| @show{(coe  '(num-expt 2 5))}
+|===
+
+- Angela says the Circles are equivalent because of the Commutative Property. Do you agree with Angela?
```

Comment:
At this point I would expect to have them create or look at the equivalent circles that don't use the function `num-expt`!